utils/callData/registrationCalls.ts

```diff
@@ -95,13 +95,21 @@ function renew(
 function multiCallRenewal(
   encodedDomains: string[],
   duration: number,
```

Comment:
may be a comment to specify that this is a duration in years (or call this durationInYears), I initially thought it was in days